python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -849,7 +849,16 @@ def get_vertex_data(self, vertex_ids=None, types=None, columns=None):
                 # FIXME: invalid columns will result in a KeyError, should a
                 # check be done here and a more PG-specific error raised?
                 df = df[[self.type_col_name] + columns]
-            return df.reset_index()
+
+            df_out = df.reset_index()
```

Comment:
I think you might want to drop the index and not have it lying around. 
```suggestion
            df_out = df.reset_index(drop=True)
```